### ISSUE TITLE
Error: Message Pagination in UI breaks iterating over message results.

### ISSUE THREAD

**Issue by jeremyjpj0916** [Graylog2/graylog2-server#6619]
I have about: 2,699,112 small messages I have stored up in a default index using a POC Graylog environment I stood up, using the UI to iterate on them like so clicking page 10 -> 20 -> 30 -> 40 -> 50 -> 60 -> ✖️ 70 ✖️  :

Fails on the button click to get to the 70th pagination page every time.

![image](https://user-images.githubusercontent.com/31913027/67178053-5b61da80-f39f-11e9-8f07-37978c9f429b.png)

Below mentions a window getting too large, but that does not make sense if the pagination was working as expected(querying the right amount of messages per page every time the same way iterating via an offset properly?). 

Error Page Pasted from the UI:

Could not execute search
There was an error executing your search. Please check your Graylog server logs for more information.

Error Message:
Unable to perform search query Result window is too large, from + size must be less than or equal to: [10000] but was [10500]. See the scroll api for a more efficient way to request large data sets. This limit can be set by changing the [index.max_result_window] index level setting.Result window is too large, from + size must be less than or equal to: [10000] but was [10500]. See the scroll api for a more efficient way to request large data sets. This limit can be set by changing the [index.max_result_window] index level setting.Result window is too large, from + size must be less than or equal to: [10000] but was [10500]. See the scroll api for a more efficient way to request large data sets. This limit can be set by changing the [index.max_result_window] index level setting.

Details:
Result window is too large, from + size must be less than or equal to: [10000] but was [10500]. See the scroll api for a more efficient way to request large data sets. This limit can be set by changing the [index.max_result_window] index level setting.
Result window is too large, from + size must be less than or equal to: [10000] but was [10500]. See the scroll api for a more efficient way to request large data sets. This limit can be set by changing the [index.max_result_window] index level setting.
Result window is too large, from + size must be less than or equal to: [10000] but was [10500]. See the scroll api for a more efficient way to request large data sets. This limit can be set by changing the [index.max_result_window] index level setting.

Search status code:
500
Search response:
cannot GET http://server.com:9000/api/search/universal/relative?query=%2A&range=432000&limit=150&offset=10350&sort=timestamp%3Adesc (500)

* Graylog Version: 3.1.2

**Comment by kroepke** [comment]
This is a know issue, closing this duplicate.

For more information, please see https://github.com/Graylog2/graylog2-server/issues/3571

tl;dr: this is an elasticsearch setting, designed to save memory, which can be tweaked by users.
